### ISSUE TITLE
Platypus: non-unique `nav` landmark

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2069]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page has `nav` landmarks which are lacking an accessible name (unique role or role/label/title combination).

---

https://dequeuniversity.com/rules/axe/4.3/landmark-unique?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Sighted Keyboard Users, Blind, Deafblind
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A
